ddterm/pref/colors.js

```
@@ -550,6 +550,41 @@ export class ColorsGroup extends PreferencesGroup {
+        });
+
+        spacer_opacity_row.set_format_value_func((_, v) => percent_format.format(v));
+
+        this.settings.bind_writable(
+            'spacer-background-opacity',
+            spacer_opacity_row,
+            'sensitive',
+            false
+        );
+
+        add_reset_button(spacer_opacity_row, this.settings, 'spacer-background-opacity', this.gettext_domain);
```

ESLint

[error] This line has a length of 110. Maximum allowed is 100.

(max-len)